docs/publishing/index.md

```diff
@@ -0,0 +1,125 @@
+
+<div class="custom-block warning"><p class="custom-block-title">WARNING</p> <p>These suggestions are not specifically tuned for publishing to a CDN or other contexts where broader application-specific optimizations will not be applied to the code available therein.</p></div>
+
+<style>
+.content__default ul {list-style:none;padding:0;}
+</style>
+
+### Do: 👍
+
+- ✅ [Do publish latest standard EcmaScript](#do-publish-latest-standard-ecmascript)
+- ✅ [Do publish standard es modules](#do-publish-standard-es-modules)
+- ✅ [Do include `"main": "index.js"` and `"module": "index.js"` in your `package.json`](#do-include-main-indexjs-and-module-indexjs-in-your-packagejson)
+- ✅ [Do export element classes](#do-export-element-classes)
+- ✅ [Do export side effects separately](#do-export-side-effects-separately)
+- ✅ [Do keep side effects out of your index.js](#do-keep-side-effects-out-of-your-indexjs)
+- ✅ [Do import 3rd party node modules with "bare" import specifiers](#do-import-3rd-party-node-modules-with-bare-import-specifiers)
+- ✅ [Do include file extensions in import specifiers](#do-include-file-extensions-in-import-specifiers)
+- _Optional_: [use `type: "module"` when possible](#optional-use-type-module)
+- _Optional_: [include typings; e.g. `*.d.ts` files](#optional-include-typings)
+- _Optional_: [include source maps](#optional-include-source-maps)
+
+### Don't: 👎
+
+- ❌ [Do not optimize](#do-not-optimize)
+- ❌ [Do not bundle](#do-not-bundle)
+- ❌ [Do not minify](#do-not-minify)
+- ❌ [Do not use `.mjs` file extensions](#do-not-use-mjs-file-extensions)
+- ❌ [Do not import polyfills](#do-not-import-polyfills)
+
+### General concept: Building is an application level concern
+
+Your users will always have the most intimate knowledge of how and where thier code will be used to their users. Give them the greatest amount of flexibility in preparing their code for those conditions by publishing your package with the agreement that building is an application level concern. Any decision that you can make in support of your users having an easier time when and if they decide to bundle is a good decision to make when preparing your package for publication. Every "do" and every "don't" that is included herein will help you in doing just that.
```

Comment:
```suggestion
Your users will always have the most intimate knowledge of how and where their code will be used to their users. Give them the greatest amount of flexibility in preparing their code for those conditions by publishing your package with the agreement that bundling is an application level concern. Any decision that you can make in support of your users having an easier time when and if they decide to bundle is a good decision to make when preparing your package for publication. Every "do" and every "don't" that is included herein will help you in doing just that.
```

"to their users" sounds slightly off, is that correct english? Should it be "for" their users?

Reply:
`by`, I think I meant `delivered` instead of `used`... 🤔